yellowbrick/regressor/influence.py

```diff
@@ -180,7 +180,8 @@ def draw(self):
         """
         # Draw a stem plot with the influence for each instance
         _, _, baseline = self.ax.stem(
-            self.distance_, linefmt=self.linefmt, markerfmt=self.markerfmt
+            self.distance_, linefmt=self.linefmt, markerfmt=self.markerfmt,
+            use_line_collection=True
```

Comment:
Great use of the `use_line_collection` parameter... this should increase performace/speed, plus  as of MPL 3.3 it will be the default setting.